### DISCUSSION TITLE
Document sdk boundaries, versioning issues, etc.

### DISCUSSION THREAD

**Discussion by AKushWarrior** [isar/isar#21] [Ideas]
Currently, the Dart interface to Isar only works if the Dart SDK is compiled from source. As far as I can tell, this is because the Isar project uses the bleeding-edge version of https://pub.dev/packages/ffi (0.3.0-nullsafety.0), which in turn relies on a version of Dart that hasn't been released yet. I feel like that deserves a mention in the README.

Also deserving of mention somewhere in the documentation, I think, is the fact that this library has been ported to null-safety and thus relies on new language features (subject to breaking change).

**Comment by simc** [comment]
Good catch, you are right. I'll update the readme

**Comment by simc** [comment]
The newest release just needs Flutter beta (nullsafety SDK) so it should be fine.
